<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertForm</class>
 <widget class="QWidget" name="ConvertForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_preprocess_algorithm">
         <item>
          <property name="text">
           <string>Nothing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EdgeDetect</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Binarize</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Stretch</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Preprocessing Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="sw_preproc_algo_settings">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="page_5"/>
          <widget class="QWidget" name="page">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Threshold</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="hs_ed_threshold">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Threshold</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="hs_binarize_threshold">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QCheckBox" name="cb_binarize_invert">
              <property name="text">
               <string>Invert</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_6">
           <layout class="QFormLayout" name="formLayout_5">
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="sb_stretch_min">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sb_stretch_max">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="cb_stretch_auto">
              <property name="text">
               <string>Automatic detection</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Quantile</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="sb_stretch_quantile">
              <property name="maximum">
               <double>0.500000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="b_apply">
         <property name="text">
          <string>apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_convert_algorithm">
         <item>
          <property name="text">
           <string>MultiLines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Squares</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Conversion settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QStackedWidget" name="sw_convert_algo_settings">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page_4">
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Sampling</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="hs_multilines_sampling">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="QCheckBox" name="cb_multiline_draw_135">
              <property name="text">
               <string>Draw 135Â° Line</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="cb_multiline_map">
              <property name="text">
               <string>Map lines to intensity</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QCheckBox" name="cb_multiline_draw_90">
              <property name="text">
               <string>Draw 90Â° Line</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="cb_multiline_draw_0">
              <property name="text">
               <string>Draw 0Â° Line</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QCheckBox" name="cb_multiline_draw_45">
              <property name="text">
               <string>Draw 45Â° Line</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>mm</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="1">
             <widget class="QSlider" name="hs_multilines_threshold">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Threshold</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>RecursionDepth</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sb_squares_recursionDepth">
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="b_convert">
         <property name="text">
          <string>convert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_stretch_auto</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_stretch_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_stretch_auto</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_stretch_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hs_multilines_sampling</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hs_multilines_threshold</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_multiline_map</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_15</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_multiline_map</sender>
   <signal>toggled(bool)</signal>
   <receiver>hs_multilines_threshold</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_multiline_map</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_14</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>656</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
